src/app/ota_image_tool.py

```diff
@@ -262,7 +262,7 @@ def show_header(args: object):
     print(f'Magic: {magic:x}')
     print(f'Total Size: {total_size}')
     print(f'Header Size: {header_size}')
-    print(f'Header TLV:')
+    print(f'Header TLV: {header_tlv}')
```

Comment:
No, `header_tlv` is printed below. I guess you can drop `f` prefix instead.

Reply:
Yes right thank you.